### ISSUE TITLE
Create down conversion for GroupNormalization-21

### ISSUE THREAD

**Issue by justinchuby** [onnx/onnx#5843]
When converters are adopting the new operator in opset 21, there will be needs for converting to lower opsets when users are trying to use runtimes that have not caught up with support.

The converter will help us adopt the latest opset without having to wait for all runtimes to catch up.

We don’t need to use the same operator (GroupNorm) when converting back, as long as the model is correct and accurate.

It is possible to use InstanceNorm and some transpose for it. For example https://github.com/microsoft/onnxscript/blob/172a81745c10467e04d9b4fdfa7829ce3321b6ce/onnxscript/function_libs/torch_lib/ops/core.py#L5850

**Comment by emecii** [comment]
Is this conversion still wanted? I recently worked on the version converter in #8425 and would be interested in taking this on. Before prototyping, should this be a 21→20 adapter using `Reshape`/`Transpose`/`InstanceNormalization`, and should dynamic shapes be supported or rejected when the group reshape cannot be represented safely?

**Comment by justinchuby** [comment]
Yes, using InstanceNormalization is the way to go. Feel free to support/not support dynamic cases depending on representability of the ops. Thanks!
